crates/control/src/role_assignment.rs

```diff
@@ -667,70 +611,26 @@ fn seen_ball_to_game_controller_ball_position(
     })
 }
 
-fn seen_ball_to_hulks_network_ball_position(
-    ball: Option<&BallPosition<Ground>>,
+fn own_ball_to_hulks_network_ball_position(
```

Comment:
what about making this and the other similar conversion functions methods of one of the types?